src/librustc_back/dynamic_lib.rs

```diff
@@ -276,7 +276,8 @@ mod dl {
             None => {
                 let mut handle = ptr::null_mut();
                 let succeeded = unsafe {
-                    GetModuleHandleExW(0 as DWORD, ptr::null(), &mut handle)
+                    let empty_flag = 0;
+                    GetModuleHandleExW(empty_flag, ptr::null(), &mut handle)
```

Comment:
Perhaps, but you can't really understand these kinds of APIs anyway unless you're reading the docs for the function at the same time, and it's not necessarily idiomatic to do this that often in Rust